tests/frontend/unit/DpClaim.spec.js

```diff
@@ -0,0 +1,74 @@
+/**
+ * (c) 2010-present DEMOS E-Partizipation GmbH.
+ *
+ * This file is part of the package demosplan,
+ * for more information see the license file.
+ *
+ * All rights reserved
+ */
+
+import { createLocalVue } from '@vue/test-utils'
+import DpClaim from '@DpJs/components/statement/DpClaim'
+import shallowMountWithGlobalMocks from '@DpJs/VueConfigLocal'
+
+describe('DpClaim', () => {
+  it('should be an object', () => {
+    expect(typeof DpClaim).toBe('object')
+  })
+
+  it('should be named DpClaim', () => {
+    expect(DpClaim.name).toBe('DpClaim')
+  })
```

Comment:
Just a side note for the future: I'm not really sure how useful these two tests are, but if we want to keep them, we should move them to a separate file and import them in the test suites.

Reply:
I can't tell neither if they are really useful. I have seen them always as a starting point since it is/wasn't always easy to load a component wich get tested here. But why do you want to move them in a separate file?

Reply:
My idea was to make these two tests more generic, so instead of writing them for each component test suite, we could just import them.